### DISCUSSION TITLE
关于媒体库和二级目录命名问题

### DISCUSSION THREAD

**Discussion by gggcpy** [jxxghp/MoviePilot#1161] [讨论]
假设不开启自动二级目录的情况下，文件夹结构建立如下：
├Media—源文件
├-------------└─电影
├------------------└─欧美电影---aaa.mkv
├------------------└─华语电影
├-------—硬链接
├-------------└─电影
├------------------└─欧美电影
├------------------└─华语电影
在目录监控的插件里配置【Media/源文件/电影/欧美电影:Media/硬链接/电影/欧美电影#link】
最终转移结果会成为：Media/硬链接/电影/欧美电影/电影/aaa.mkv
即最终会强制嵌套一个LIBRARY_MOVIE_NAME变量的名字。
不知道是否可以不强制创建LIBRARY_PATH变量的文件夹。
或者LIBRARY_PATH下的分类是否可以设置成用户自己添加一个或多个并命名。这样就可以在不启用自动二级分类的情况下，用户根据自己的需要设定多个一级目录来实现二级分类的功能。
以上。非常感谢。

**Comment by cng10001** [comment]
![image](https://github.com/jxxghp/MoviePilot/assets/115156545/89a28dbf-54ce-4fdd-ae4b-ab1dbcb0a107)
这个应该解决了吧
